src/main/java/Main.java

```diff
@@ -1,21 +1,35 @@
 import mysticalAnimal.Dragon;
 import mysticalAnimal.DragonType;
```

Comment:
nit: fun fact that it's recommended to use only lower case when naming the packages. 
`mysticalanimal` is the way to go